src/widgets/conditions/conditions-detailed-summary.css

```
@@ -0,0 +1,108 @@
+:global(.omrs-breakpoint-gt-phone) .conditionSummary {
+  display: flex;
+  justify-content: center;
+}
+
+:global(.omrs-breakpoint-lt-tablet) .conditionSummary {
+  padding: 0.25rem 0.5rem 0 0.5rem;
+}
+
+.conditionTable {
+  width: 100%;
+}
+
+.conditionTable thead {
+  color: var(--omrs-color-ink-medium-contrast);
+  border-bottom: 1px solid var(--omrs-color-bg-low-contrast);
```

convert this to use rem
```suggestion
  border-bottom: 0.0625rem solid var(--omrs-color-bg-low-contrast);
```

---

Done